index.sh

```diff
@@ -1,11 +1,4 @@
-s=$(which mighty-batch)
-if [ "$s" = "" ];
-then
-    npm install -g mighty-batch
-    cd tools
-    npm install
-    cd ..    
-fi
-tar -zxf outdoors_posts.tar.gz
-mighty-batch --threads 1 --workers 2 --host localhost --json outdoors_posts.json --property Paragraphs
-node tools/load.js --files vectors/outdoors_posts.json/ --name outdoors
+#!/usr/bin/env bash
+
+docker build ./tools --tag=mighty-indexer
+docker run --rm -it --network mqsnet mighty-indexer:latest ./index.sh $*
```

Comment:
`--network mqsnet` - enable the process to access qdrant for uploading, which is now protected from external access

Reply:
`./index.sh $*` will simply forward attributes to the original script